examples/social-share.amp.html

```diff
@@ -1,109 +1,182 @@
-<!doctype html>
+<!DOCTYPE html>
```

Comment:
What are the actual changes on this file? If it is just the result of running prettify, can you revert?

Reply:
There are two changes (other than the result of prettify).

One is to change the experiment name:
https://github.com/ampproject/amphtml/pull/29084/files#diff-061782f8e93cebf19da60de906338542R25

The other is to change the version of amp-component since we renamed it from 0.2 -> 1.0.
https://github.com/ampproject/amphtml/pull/29084/files#diff-061782f8e93cebf19da60de906338542R30

I'm thinking we should also keep the prettify changes since otherwise it'll just get updated by the next person who changes this file.  Let me know if you disagree!

Reply:
Up to you. You can revert the formatting changes, reapply the two lines you linked, and save without auto-formatting in VS Code with `Cmd K` + `S`. If you don't want to revert the whole file, please at least reshrink the boilerplate to one line. :)

Reply:
Makes sense, I've gone ahead and reverted the whole file to make it easier to see what's going on in the review.